pkg/images/types/digest.go

```diff
@@ -32,7 +34,7 @@ func (d *Digest) Algorithm() string {
 	if d == nil {
 		return ""
 	}
-	return d.algorithm + ":" + d.hash
```

Comment:
What!

Reply:
looks like this is never used, so it was never caught before :(